components/Chips/src/MDCChipView.m

```diff
@@ -136,6 +136,7 @@ - (void)commonMDCChipViewInit {
   _minimumSize = kMDCChipMinimumSizeDefault;
   self.isAccessibilityElement = YES;
   _mdc_legacyFontScaling = YES;
+  _adjustsFontForContentSizeCategoryWhenScaledFontIsUnavailable = YES;
```

Comment:
Given that we are renaming mdc_legacyFontScaling I would expect us to simply forward `mdc_legacyFontScaling`'s getter/setter to `adjustsFontForContentSizeCategoryWhenScaledFontIsUnavailable` and to update the logic below accordingly.

Reply:
Do you mean directly backing `mdc_legacyFontScaling` property by `_adjustsFontForContentSizeCategoryWhenScaledFontIsUnavailable` instance?

Reply:
Yep.

Reply:
Jeff, I thought you said these fixes had to be a deprecation (deletion next release) plus new API. Did that change?

Reply:
Addressed in 23b88d5ecf3a3e0a4a84e76d7d2e40afc71dfb39.